src/app/dataset-view/additional-components/overview-component/components/readme-section/readme-section.component.html

```diff
@@ -0,0 +1,77 @@
+<div
+    id="file-readme-dataset-template-md"
+    class="file my-4"
+    *ngIf="currentState?.overview?.metadata?.currentReadme"
+>
+    <div class="file-header d-flex align-items-center">
+        <div
+            class="file-info pr-4 d-flex flex-order-1 flex-auto align-items-center"
+        >
+            <span class="mr-2">
+                <svg-icon name="code-square"></svg-icon>
+            </span>
+            <h2 class="box-title align-items-center m-0">
+                <a
+                    href="#readme"
+                    data-view-component="true"
+                    class="link--primary"
+                    >README.md</a
+                >
+            </h2>
+        </div>
+        <button
+            class="starred rounded-2 btn-sm btn btnGroup-item custom-edit d-flex justify-content-center align-items-center"
+            aria-label="Edit this file"
+            data-test-id="toggleReadme"
+            data-hotkey="e"
+            data-disable-with=""
+            (click)="toggleReadmeView()"
+        >
+            <svg-icon name="pencil"></svg-icon>
+        </button>
+    </div>
+    <div
+        *ngIf="isMarkdownEditView"
+        class="edit-title d-flex align-items-center justify-content-between"
+    >
+        <div class="left-part">
+            <button
+                class="edit-button"
+                [class.active]="isEditMode"
+                (click)="toggleEditMode()"
+            >
+                Edit
+            </button>
+            <button
+                class="edit-button"
+                [class.active]="!isEditMode"
+                (click)="toggleEditMode()"
+            >
+                Preview
+            </button>
+        </div>
+        <div class="right-part d-flex">
+            <button class="button-changes mr-4" (click)="onCancelChanges()">
+                Cancel changes
+            </button>
+            <button
+                class="button-changes"
+                [disabled]="!readmeChanged"
+                (click)="commitChanges()"
+            >
+                Commit changes...
```

Comment:
Save ? The term "Commit" is a bit internal.

Reply:
Save, fixed